src/modules/create.js

```diff
@@ -1,5 +1,8 @@
-export default function create(array) {
-  array
-      += array;
-  return array;
+const list = document.querySelector('.leaderboard');
+
+export default function create(array = []) {
+  list.innerHTML = '';
+  for (let i = 0; i < array.length; i += 1) {
+    list.innerHTML += `<li>${array[i].index} type="checkbox"><input data-id="${array[i].index}" class="taskEdit" type="text" value ="${array[i].description}"></li>`;
+  }
 }
```

Comment:
- [ ] Great job so far :clap: but I think it will be better if you can use `arrow functions` syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Thank you! Those were placeholder functions that are not necessary for the functioning of the program so I removed them altogether.